docs/predictive-analytics/model-retraining/sink.json

```diff
@@ -0,0 +1,31 @@
+{
+    "connector.class": "MongoDbAtlasSink",
```

Comment:
@daveklein for readability, would you be able to please align all these lines so the `:` are in a line?

Reply:
Done